cms/users/tests/test_viewsets.py

```diff
@@ -0,0 +1,58 @@
+from django.test import TestCase
+from django.urls import reverse
+from wagtail.test.utils import WagtailTestUtils
+
+from cms.users.tests.factories import UserFactory
+from cms.users.viewsets import user_chooser_viewset
+
+
+class TestUserChooserViewSet(WagtailTestUtils, TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.chooser_url = user_chooser_viewset.widget_class().get_chooser_modal_url()
+        cls.chooser_results_url = reverse(user_chooser_viewset.get_url_name("choose_results"))
+
+        cls.user_without_name = UserFactory(username="noname", first_name="", last_name="", is_active=False)
+        cls.user_with_first_name = UserFactory(username="first_name_user", first_name="Data", last_name="")
+        cls.user_with_last_name = UserFactory(username="last_name_user", first_name="", last_name="La Forge ")
```

Comment:
Nit-pick: Dangling space

```suggestion
        cls.user_with_last_name = UserFactory(username="last_name_user", first_name="", last_name="La Forge")
```

Reply:
Good catch. And that is on purpose as `get_user_display_name` does `.strip()`. Granted it is a core method which should be tested, but I left it on purpose